src/Collection/Collection.router.ts

```diff
@@ -146,7 +153,15 @@ export class CollectionRouter extends Router {
       collectionAPI.fetchCollections(),
     ])
 
-    return Bridge.consolidateCollections(dbCollections, remoteCollections)
+    const consolidatedCollections = await Bridge.consolidateCollections(
+      dbCollections,
+      remoteCollections
+    )
+
+    // Build the full collection
+    return consolidatedCollections.map((collection) =>
+      toFullCollection(collection)
+    )
   }
 
   async getAddressCollections(req: AuthRequest) {
```

Comment:
Maybe we can go the `Item.router.ts` route (heh) here too and add return types to these methods, no?